docs/resources/itinerary-managment.md

```diff
@@ -151,3 +155,61 @@ The result? You can probably guess it. :)
 }
 ```
 
+## Find a city by keywords
```

Comment:
I think it would be interesting to explain why a developer would use the City Search API instead of the [Airport and City Search](https://developers.amadeus.com/self-service/category/air/api-doc/airport-and-city-search) or the other way around. They  both look similar and might confuse developers.